src/ImageSharp.Web.Providers.Azure/Providers/AzureBlobStorageImageProvider.cs

```diff
@@ -29,11 +30,6 @@ public class AzureBlobStorageImageProvider : IImageProvider
         /// </summary>
         private readonly CloudStorageAccount storageAccount;
 
-        /// <summary>
-        /// The container in the blob service.
-        /// </summary>
-        private readonly CloudBlobContainer container;
```

Comment:
Make this a `Dictionary<string, CloudBlobContainer>` which we create on startup.